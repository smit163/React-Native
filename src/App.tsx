/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 */

import React, { Component } from 'react';
import { AppContainer } from './config/navigation';


class App extends Component<Props> {
  constructor(props) {
    super(props)
  }
  render() {
    return (    
          <AppContainer />   
      
    ) 
  }
}


export default App;

