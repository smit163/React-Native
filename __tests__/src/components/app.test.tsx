import 'react-native'
import React from 'react';
import { ApolloProvider } from 'react-apollo';
import App from '../../../src/App'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import { createMockClient } from 'mock-apollo-client';
import toJson from 'enzyme-to-json';

it('renders correctly11', () => {
    const client = createMockClient();
    const tree = renderer.create(<ApolloProvider client={client}><App /></ApolloProvider>);
    expect(toJson(tree)).toMatchSnapshot();
});