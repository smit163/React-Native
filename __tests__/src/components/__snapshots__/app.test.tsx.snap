// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly11 1`] = `undefined`;
